exercises/simple-cipher/src/test/java/SimpleCipherStepThreeTest.java

```diff
@@ -0,0 +1,61 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Step 3: Generate random key if key isn't specified. Check key is right format
+ */
+public class SimpleCipherStepThreeTest {
+
+    @Ignore
+    @Test
+    public void cipherKeyIsMadeOfLetters() {
+        assertTrue(new Cipher().getKey().matches("[a-z]+"));
+    }
+
+    @Ignore
+    @Test
+    public void defaultCipherKeyIs100Characters() {
+        assertEquals(100, new Cipher().getKey().length());
+    }
+
+    @Ignore
+    @Test
+    public void cipherKeysAreRandomlyGenerated() {
+        String newKey = new Cipher().getKey();
+        assertFalse("Cipher constructor without argument should generate a random key. No two calls to the" +
+                " constructor should generate the same key. Two calls to the constructor " +
+                "both returned key: " + newKey, newKey.equals(new Cipher().getKey()));
+    }
+
+    @Ignore
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
This is slightly out of scope for the work you're doing, @FridaTveit, but if you're so inclined...

... since we've upgraded to JUnit 4.12, we've been migrating to using JUnit's [`ExpectedException`](http://junit.org/junit4/javadoc/4.12/org/junit/rules/ExpectedException.html) `@Rule` feature for this situation.

If you so choose, since we're making changes, here could convert these to use that notation.